security/libaes_siv/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+# NOTE: Doesn't work with cmake 1.1 PortGroup
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.0
+
+github.setup        dfoxfranke libaes_siv 1.0.0 v
+revision            0
+categories          security devel
+maintainers         {fwright.net:fw @fhgwright} openmaintainer
+
+description         A C implementation of AES-SIV
+
+long_description    This is an RFC5297-compliant C implementation of AES-SIV, \
+                    written by Daniel Franke on behalf of Akamai Technologies \
+                    and published under the Apache License (v2.0). It uses \
+                    OpenSSL for the underlying AES and CMAC implementations \
+                    and follows a similar interface style.
+
+license             apache-2
+platforms           darwin
+
+github.tarball_from releases
+master_sites        https://github.com/${github.author}/${name}/archive
+distname            v${github.version}
+worksrcdir          ${name}-${github.version}
+
+checksums           rmd160  c9e247100cb462650a16bd18aee05677adca8751 \
+                    sha256  a974406dc3e3addbecdc0f0c1e2050bede1c7b4394595e9536094a87dac617c9 \
+                    size    18829
+
+default_variants    +doc
+
+variant doc description {Build manpages (depends on asciidoc)} {
+    depends_build-append    port:asciidoc
+}
+if {![variant_isset doc]} {
+    configure.args-append   -DDISABLE_DOCS=1
+}
```

Comment:
Is the documentation so large that it's worth having a separate variant without it?

Reply:
No, but _building_ the documentation adds (recursively) 25 more dependencies.

Reply:
That's not a big deal. Most users download binary packages. I wouldn't worry about it.

Reply:
And the binary package will include the docs, since that's the default variant.

This is exactly the same doc-related setup as is used in the `ntpsec` port.  And that isn't even my doing, since it was set up that way before I took over as the maintainer. :-)

Reply:
Given this, it's sort of pointless (in both cases) to provide the variant. No one is ever going to use it.

Reply:
Well, I'm a fan of flexibility, and not a fan of dependency bloat.  The main downside of variants is the expansion of the test matrix, but since I'm the one doing the testing, it shouldn't be someone else's place to complain about it.

Reply:
In the absence of further comments, I'm marking this resolved.  The other issues have already been addressed (and so marked).